notebooks/5094_harmonisation.py

```diff
@@ -115,37 +128,68 @@
 
 # %%
 history_for_gridding_harmonisation = HISTORY_HARMONISATION_DB.load(pix.ismatch(purpose="gridding_emissions"))
-history_for_gridding_harmonisation
+# history_for_gridding_harmonisation
 
 # %%
 # Add history (zero) history for the new CDR sectors
 
 # create a template (all regions, same 'historical' scenario, same unit as other CO2, for the same number of years
-history_for_gridding_harmonisation_template = history_for_gridding_harmonisation.loc[pix.ismatch(variable=["Emissions|CO2|Energy Sector"])]
-history_for_gridding_harmonisation_template = history_for_gridding_harmonisation_template.pix.assign(model="Synthetic") # it is not CEDS, we just add it ourselves
+history_for_gridding_harmonisation_template = history_for_gridding_harmonisation.loc[
+    pix.ismatch(variable=["Emissions|CO2|Energy Sector"])
+]
+history_for_gridding_harmonisation_template = history_for_gridding_harmonisation_template.pix.assign(
+    model="Synthetic"
+)  # it is not CEDS, we just add it ourselves
 template_years = [col for col in history_for_gridding_harmonisation_template.columns if isinstance(col, int)]
-history_for_gridding_harmonisation_template.loc[:, template_years] = 0.0 # data is zero
+history_for_gridding_harmonisation_template.loc[:, template_years] = 0.0  # data is zero
 
 # create the two extra sectors
-history_for_gridding_harmonisation_BECCS = history_for_gridding_harmonisation_template.pix.assign(variable="Carbon Removal|CO2|BECCS")
-history_for_gridding_harmonisation_nonLandCDR = history_for_gridding_harmonisation_template.pix.assign(variable="Carbon Removal|CO2|Other non-Land CDR")
+history_for_gridding_harmonisation_BECCS = history_for_gridding_harmonisation_template.pix.assign(
+    variable="Carbon Removal|CO2|BECCS"
+)
+history_for_gridding_harmonisation_nonLandCDR = history_for_gridding_harmonisation_template.pix.assign(
+    variable="Carbon Removal|CO2|Other non-Land CDR"
+)
 
 # concatenate with the historical gridding data
 
-history_for_gridding_harmonisation = pd.concat([
-    history_for_gridding_harmonisation,
-    history_for_gridding_harmonisation_BECCS,
-    history_for_gridding_harmonisation_nonLandCDR
-])
+history_for_gridding_harmonisation = pd.concat(
+    [
+        history_for_gridding_harmonisation,
+        history_for_gridding_harmonisation_BECCS,
+        history_for_gridding_harmonisation_nonLandCDR,
+    ]
+)
 
-history_for_gridding_harmonisation
+# history_for_gridding_harmonisation
 
 # %%
 history_for_global_workflow_harmonisation = HISTORY_HARMONISATION_DB.load(
     pix.ismatch(purpose="global_workflow_emissions")
 )
 # history_for_global_workflow_harmonisation
 
+# %%
+# Handling CDR at global level IS IT NEEDED???
```

Comment:
Good note. I need to get back to this one. 
Probably you do the right thing.

The global_workflow should be updated after the regional harmonization is done, and then aggregate that. (for CO2|AFOLU that would now 'break' a bit, but OK we can fix that)

I think it is act
(The question is if we can keep it split out from Fossil; update the reaggregated only after harmonization?)